includes/polling/ports/port-poe.inc.php

```diff
@@ -59,24 +59,4 @@
         data_update($device, 'poe', $tags, $fields);
         echo 'PoE(IOS) ';
     }//end if
-} else {
-    //This is the legacy code, to be tested against devices. This code looks terribly broken. There is
-    //most probably no device that can show anything out of this ...
-
-    if ($this_port['dot3StatsIndex'] && $port['ifType'] == 'ethernetCsmacd') {
```

Comment:
This condition is really broken. A lot of devices do have dot3StatsIndex set and will then create a completely empty RRD ...